### PR TITLE
set nav cmds instead of angles in nav manual mode

### PR DESCRIPTION
**Pull request by EwoudSmeur**
I think having dedicated variables for the commands in nav manual mode makes more sense than abusing the nav angle setpoints for this purpose. 
This can also resolve bugs when using this mode: nav_heading is for instance also set (as a BFP angle) by guidance_h_nav_enter()

### CODE REVIEW

**Comment by podhrmic** [comment]
@EwoudSmeur I must have missed something, but what is NAV MANUAL mode?

**Comment by EwoudSmeur** [comment]
It is a nav mode you can set from the flight plan, like the nav mode attitude. It allows the user to set the stabilization commands from the flight plan. 

It is useful for helicopters, as they need a few seconds to spin up, in which you may not want a controller to run. So you can set the stabilization commands to zero while the rotor starts spinning.

**Comment by podhrmic** [comment]
ah cool, in that case it looks good to me.

**Comment by flixr** [comment]
Basically looks good to me, but some doxygen comments on what the vars are used for would be nice.
And/or rename them to something like `nav_manual_roll` or so to make this more clear?

**Comment by EwoudSmeur** [comment]
Something like this? 
I think the naming is clear this way, it is a command set from nav. The naming of HORIZONTAL_MODE_MANUAL is more confusing in my eyes.

File: sw/airborne/firmwares/rotorcraft/navigation.c [hunk 1]
```diff
@@ -482,16 +486,20 @@ void nav_home(void)
...
- * @param[in] yaw The angle in radians (float)
+ * @param[in] roll command in pprz scale (float)
+ * @param[in] pitch command in pprz scale (float)
+ * @param[in] yaw command in pprz scale (float)
+ *
+ * This function allows to directly set commands from the flight plan,
+ * if in nav_manual mode.
+ * This is for instance useful for helicopters during the spinup
  */
 void nav_set_manual(float roll, float pitch, float yaw)
```
Comment (flixr) [inline sw/airborne/firmwares/rotorcraft/navigation.c]: why have roll, pitch, yaw as floats here when the commands are always in int32_t anyway?

**Pull request merged**